c2cgeoportal/scripts/c2ctool.py

```diff
@@ -155,8 +161,9 @@ class C2cTool:
 
     def print_step(self, step, intro="To continue type:"):
         print(intro)
-        print(_colorize("%s upgrade %s %s --step %i", YELLOW) % (
+        print(_colorize("%s upgrade %s %s %s --step %i", YELLOW) % (
             "%s/c2ctool" % self.venv_bin,
+            '--windows WINDOWS' if self.options.windows is not None else '',
```

Comment:
Your right buts it's because the windows argument is wring configured, we should add 
`action="store_true",`